### DISCUSSION TITLE
Job outputs API

### DISCUSSION THREAD

**Discussion by vaind** [community/community#32540] [Apps, API and Webhooks] [answered]
Is there any way to get job/workflow outputs (`::set-output name={name}::{value}`)? Ideally a rest API.

**Comment by airtower-luna** [comment, marked as answer]
There isn't an API like that. If you want to make data available outside the workflow you might want to [upload it as an *artifact*](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts).

A simple way to create such an artifact might be to use [`toJSON()`](https://docs.github.com/en/actions/learn-github-actions/expressions#tojson) on the context (or part of a context) you want.
